### PR TITLE
Stop running the test suite on Symfony 2

### PR DESCRIPTION
**Pull request by greg0ire**
We should drop support for it. Also, if we get a bug with Symfony, it
will not get fixes on Symfony 2, and we cannot have that.

### CODE REVIEW

File: config/projects.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ admin-bundle:
     3.x:
       php: ['5.6', '7.0', '7.1', '7.2']
       versions:
-        symfony: ['2.8', '3.3', '3.4']
+        symfony: ['3.3', '3.4']
```
Comment (core23) [inline config/projects.yml]: There is no active support for symfony 3.3 anymore: https://symfony.com/roadmap/3.3

Comment (OskarStark) [reply config/projects.yml]: Lets do it in a following PR, and see if everything will work here 😄

**Comment by OskarStark** [comment]
Thank you!

File: config/projects.yml (hunk 1 above)
Comment (OskarStark) [reply config/projects.yml]: already done in this PR 👍

**Pull request merged**